Lesson 12/src/index.js

```diff
@@ -0,0 +1,108 @@
+// 1
+
+let number = 0
+
+while (number <= 20) {
+  console.log(number)
+  if (number === 15) {
+    break
+  }
+  number += 1
+  // if (a === 10) {
+  //   continue
+  // }
+}
+
+// 2
+function findEvenNumber(number1) {
+  if (number1 % 2 === 0) {
+    return true
+  } return false
+}
+findEvenNumber(16)
+findEvenNumber(7)
+
+// 3
+
+const users = [
+  { name: 'Alice', age: 25 },
+  { name: 'Bob', age: 30 },
+  { name: 'Charlie', age: 35 }
+]
+
+users.forEach((element) => {
+  console.log(`Имя пользователя: ${element.name}, Возраст: 
+    ${element.age}`)
+})
+
+// 4
+
+const products = [
+  { name: 'Laptop', price: 1000 },
+  { name: 'Phone', price: 500 },
+  { name: 'Tablet', price: 300 }
+]
+
+const productsWithNewPrice = products.map((element) => ({ ...element, price: element.price * 1.1 }))
+console.log(productsWithNewPrice)
+
+// 5
+
+const tasks = [
+  { title: 'Task 1', completed: true },
+  { title: 'Task 2', completed: false },
+  { title: 'Task 3', completed: true }
+]
+
+const newTasks = tasks.filter((element) => element.completed !== false)
+console.log(newTasks)
```

Comment:
```suggestion
const completedTasks = tasks.filter((task) => task.completed)

console.log(completedTasks)
```

1. названия переменных не `a`, `b` - уже хорошо. следующий шаг - называй тем, что содержит переменная. `newTasks` слишком общее название и не отражает сути. из названия `completedTasks` сразу очевидно, какие задачи

3. `filter` ожидает true / false чтобы оставить или удалить элемент. в объектах задач, `completed` как раз имеет значения true / false - это можно сразу и вернуть как результат